<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>欢迎使用软酷洗衣</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="username">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>username</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>810</width>
      <height>541</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>订单管理</string>
     </attribute>
     <widget class="QStackedWidget" name="page_order">
      <property name="geometry">
       <rect>
        <x>98</x>
        <y>-1</y>
        <width>711</width>
        <height>511</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page01">
       <widget class="QRadioButton" name="radioButton_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>200</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>鞋子</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>120</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>杀菌   1元/件</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>40</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>洗衣服务：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_phone">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>290</y>
          <width>261</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>120</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>大件</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>80</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>普通衣服</string>
        </property>
       </widget>
       <widget class="QLabel" name="money_total">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>350</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>总金额：0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>290</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>手机号：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>330</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>备注：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>160</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>烘干   1元/件</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_confirm">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>390</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>确认下单</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>240</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>干洗</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>洗衣类型：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>80</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>标准洗涤   3元/件</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>40</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>衣物数量：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>80</y>
          <width>81</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>160</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>床上用品</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="edit_extra">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>330</y>
          <width>261</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page02">
       <widget class="QTableView" name="orderTable">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>701</width>
          <height>461</height>
         </rect>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>95</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>40</number>
        </attribute>
       </widget>
       <widget class="QPushButton" name="btn_right">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>13</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_left">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>13</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="search_order">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>251</width>
          <height>24</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>请输入搜索内容</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_search_order">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>10</y>
          <width>61</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
       <widget class="QLabel" name="num_search">
        <property name="geometry">
         <rect>
          <x>519</x>
          <y>14</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_history">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>61</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>查看</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_order">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>61</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>下单</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>会员模块</string>
     </attribute>
     <widget class="QTableView" name="table_member">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>511</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>190</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>40</number>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>店面管理模块</string>
     </attribute>
     <widget class="QTableView" name="table_manage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>511</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>130</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>40</number>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
